api/src/main/java/org/openmrs/module/patientmatching/PatientMatchingActivator.java

```diff
@@ -181,7 +176,7 @@ public void willStart() {
 	public void contextRefreshed() {
 		log.info("Starting Patient Matching Module");
 		
-		try {
+		/*try {
```

Comment:
I just commented out this code because in the future we will need it so that we populate the scratch table once and keep it updated with incremental changes

Reply:
Do you mind putting an actual TODO comment in the code?

Reply:
I'm not sure if it's necessary, there is no guarantee this code will be needed but there is a small chance so I didn't want to remove it.

Reply:
Putting the TODO would explain why you did against our convention as you can see it here https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode 😊